src/main/java/com/troubleshooting/troubleshootingtool/ContainerDataRetrieval.java

```diff
@@ -0,0 +1,69 @@
+package com.troubleshooting.troubleshootingtool;
+
+import com.troubleshooting.model.ConjurLogModel;
+import com.troubleshooting.model.EnvModel;
+
+import com.troubleshooting.model.EnvsModel;
+import com.troubleshooting.model.LogsModel;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+// Retrieves the data returned from the Docker Client and packs the data in the appropriate data models
+@Component
+public class ContainerDataRetrieval implements IDataRetrieval {
+    ContainerDataAccess access = new ContainerDataAccess();
+
+    ConjurLogModel log = new ConjurLogModel();
+    LogsModel logs = new LogsModel();
+
+    EnvModel env = new EnvModel();
+    EnvsModel envs = new EnvsModel();
+
+    //	Extract Logs
+    @Override
+    public LogsModel getData(String containerID) {
```

Comment:
what this method do? does it get data or extract logs? name os misleading

Reply:
updated to getLogs()

Reply:
updated to `getLogs()`